.github/workflows/ci.yaml

```diff
@@ -354,6 +365,8 @@ jobs:
       FULL_BUILD_DISABLED: true
       JEST_TEST_PATTERN: packages/cactus-api-client/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts
       JEST_TEST_RUNNER_DISABLED: false
+      JEST_TEST_COVERAGE_PATH: ./code-coverage-ts/cactus-api-client
+      JEST_TEST_CODE_COVERAGE_ENABLED: false
```

Comment:
@aldousalvarez Where is this flag is being implemented for it to be effective?

Reply:
@petermetz It is used on the ci.sh file which changes the command of the yarn jest that uses the additional --coverage parameter, so that we can generate a coverage report for that specific package since in the ci.sh we have there has a command that will run jest